src/classes/frDonation.cls

```diff
@@ -41,8 +41,9 @@
 public class frDonation extends frModel implements frSyncable {
     public static List<frMapping__c> mappings {
         get {
+            String objType = TYPE;
```

Comment:
It looks like this change is not doing anything now, can we remove this variable and restore the query to how it was previously?

Reply:
TYPE is now dynamically being set to the correct obj!

Reply:
Sorry i meant the `objType` variable looks like it is unnecessary, as we're just re-assigning a variable

Reply:
Hi team, can this variable be removed?